MIGRATION_GUIDE.md

```
@@ -3,6 +3,49 @@
+#### Rollout summary
+
+| Behavior | This release (5.4.0) | Future major release |
+|---|---|---|
+| `klaviyo_automatic_push_open_tracking` | Opt-in (default off) | Default on (opt-out) |
+| `klaviyo_automatic_push_token_forwarding` | Opt-in (default off) | Default on (opt-out) |
+
+> **Note:** If you enable `klaviyo_automatic_push_token_forwarding`, the SDK will register an
+> anonymous profile with Klaviyo as soon as a device token is obtained, before any call to
+> `setProfile`, `setEmail`, or `setPhoneNumber`. This is expected behavior — the anonymous
+> profile is merged with the identified profile once you provide identifiers.
+> See the [Anonymous Tracking Notice](#anonymous-tracking-notice) in the README.
```

markdownlint-cli2 (0.23.0)

[warning] Link fragments should be valid

(MD051, link-fragments)